tst/regression/test_suites/sparse_advection/sparse_advection.py

```diff
@@ -57,7 +57,7 @@ def Analyse(self, parameters):
                 + "/tst/regression/gold_standard/sparse_fake.out0.final.phdf",
             ],
             one=True,
-            tol=1e-6,
+            tol=2e-6,
```

Comment:
Because it fails with `1.e-6` ;). 

This test is comparing the sparse solution to the same problem run with fields allocated everywhere. I think there is no expectation that you will agree at a level better than the sparse threshold (1.e-6), just of order the sparse threshold. I didn't see how close to 1.e-6 I could get, it may be that the tolerance could be made to be 1.1e6 or something.